pkg/apis/nmstate/v1alpha1/nodenetworkconfigurationenactment_types.go

```diff
@@ -37,12 +37,15 @@ const (
 	NodeNetworkConfigurationEnactmentConditionAvailable   ConditionType = "Available"
 	NodeNetworkConfigurationEnactmentConditionFailing     ConditionType = "Failing"
 	NodeNetworkConfigurationEnactmentConditionProgressing ConditionType = "Progressing"
+	NodeNetworkConfigurationEnactmentConditionMatching    ConditionType = "Matching"
 )
 
 const (
-	NodeNetworkConfigurationEnactmentConditionFailedToConfigure        ConditionReason = "FailedToConfigure"
-	NodeNetworkConfigurationEnactmentConditionSuccessfullyConfigured   ConditionReason = "SuccessfullyConfigured"
-	NodeNetworkConfigurationEnactmentConditionConfigurationProgressing ConditionReason = "ConfigurationProgressing"
+	NodeNetworkConfigurationEnactmentConditionFailedToConfigure                 ConditionReason = "FailedToConfigure"
+	NodeNetworkConfigurationEnactmentConditionSuccessfullyConfigured            ConditionReason = "SuccessfullyConfigured"
+	NodeNetworkConfigurationEnactmentConditionConfigurationProgressing          ConditionReason = "ConfigurationProgressing"
+	NodeNetworkConfigurationEnactmentConditionNodeSelectorNotMatching           ConditionReason = "NodeSelectorNotMatching"
+	NodeNetworkConfigurationEnactmentConditionNodeSelectorAllConditionsMatching ConditionReason = "AllConditionsMatching"
```

Comment:
What do you mean by AllConditions? We should make it AllSelectors in case any selectors are used or just Matching. Conditions may be confused with status conditions.